packages/data/src/plugins/actions.ts

```diff
@@ -348,6 +349,36 @@ export function setRecommendedPlugins(
 	};
 }
 
+const SUPPORTED_TYPES = [ 'payments' ];
+export function* dismissRecommendedPlugins( type: RecommendedTypes ) {
```

Comment:
Is there a good way to handle the typescript return type with these generator functions? My linting is going into 🎄 mode on this file.

Reply:
I am actually not a 100% sure, as I don't think we set the return types explicitly anywhere. Also from what I gather that whenever you call a function with `yield someFunction()` it actually returns an `unknown` type, so you did have to set the type on the returned value. 
Usually I would go see what Gutenberg does for this, but their's is still vanilla JS.